ts/packages/node/src/index.ts

```diff
@@ -20,6 +20,7 @@ export * from './stores/Etcd';
 export * from './stores/SQLite';
 export * from './stores/MySQL';
 export * from './stores/MariaDB';
+export * from './stores/MongoDb';
```

Comment:
Please keep the naming consistent with other stores:
```suggestion
export * from './stores/MongoDB';
```